### PR TITLE
More AWS cluster create fields

### PR DESCRIPTION
**Pull request by brennancheung**
Adding more fields to the AWS cluster create wizard.

### CODE REVIEW

File: src/app/plugins/kubernetes/components/infrastructure/ClusterDomainPicklist.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  id: PropTypes.string.isRequired,
+  cloudProviderId: PropTypes.string,
+  cloudProviderRegionId: PropTypes.string,
+  initialValue: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  onChange: PropTypes.func,
+  ...ValidatedFormInputPropTypes,
+}
+
+export default compose(
+  withInfoTooltip,
```
Comment (xantorres) [inline src/app/plugins/kubernetes/components/infrastructure/ClusterDomainPicklist.js]: This is not needed, as `PicklistField` already uses `Tooltip`

File: src/app/plugins/kubernetes/components/infrastructure/AwsClusterSshKeyPicklist.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  id: PropTypes.string.isRequired,
+  cloudProviderId: PropTypes.string,
+  cloudProviderRegionId: PropTypes.string,
+  initialValue: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  onChange: PropTypes.func,
+  ...ValidatedFormInputPropTypes,
+}
+
+export default compose(
+  withInfoTooltip,
```
Comment (xantorres) [inline src/app/plugins/kubernetes/components/infrastructure/AwsClusterSshKeyPicklist.js]: Not needed

File: src/app/plugins/kubernetes/components/infrastructure/AddAwsClusterPage.js [hunk 3]
```diff
@@ -208,18 +437,31 @@ const AddAwsClusterPage = () => {
...
-                  {/* Disable workloads on master nodes */}
-                  {/* Enable spot workers */}
-                  {/* Percent of worker nodes as spot instances */}
-                  {/* Spot price */}
+                  {/* TODO */}
+                </ValidatedForm>
+              </FormWrapper>
+            </WizardStep>
+
+            <WizardStep stepId="network" label="Network Info">
```
Comment (xantorres) [inline src/app/plugins/kubernetes/components/infrastructure/AddAwsClusterPage.js]: This file is getting huge, wouldn't be better to refactor out the steps contents into separate files, and put everything in a `/AddAwsCluster` folder or something?

Comment (brennancheung) [reply src/app/plugins/kubernetes/components/infrastructure/AddAwsClusterPage.js]: We should probably refactor and move a bunch of the infrastructure stuff into subfolders.  Let's save that for a future PR.  For now, it's easier to work with when it is all in one place.

File: src/app/plugins/kubernetes/components/infrastructure/ClusterDomainPicklist.js (hunk 1 above)
Comment (brennancheung) [reply src/app/plugins/kubernetes/components/infrastructure/ClusterDomainPicklist.js]: I'll make the changes here and in the other spots.

**Pull request merged**